ActiveTheme.php

```diff
@@ -42,8 +43,11 @@ class ActiveTheme
      * @param array                           $themes
      * @param Helper\DeviceDetectionInterface $deviceDetection
      */
-    public function __construct($name, array $themes = array(), DeviceDetectionInterface $deviceDetection = null)
+    public function __construct($name, array $themes = array(), DeviceDetectionInterface $deviceDetection = null, ThemeLocator $themeLocator)
```

Comment:
this change breaks the tests:
https://travis-ci.org/liip/LiipThemeBundle/jobs/91007259#L219

either update the tests or maybe its better anyway to make this parameter optional ..